cmd/lncli/walletrpc_active.go

```diff
@@ -340,3 +341,61 @@ func listSweeps(ctx *cli.Context) error {
 
 	return nil
 }
+
+var labelTxCommand = cli.Command{
+	Name:      "labeltx",
+	Usage:     "adds a label to a transaction",
+	ArgsUsage: "txid",
+	Description: `
+	Add a label to a transaction. If the transaction already has a label, 
+	this call will fail unless the overwrite option is set. The label is 
+	limited to 500 characters.
+	`,
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name: "label",
+			Usage: "the label to add to the tx, limited to 500 " +
+				"chars",
+		},
+		cli.BoolFlag{
+			Name:  "overwrite",
+			Usage: "set to overwrite existing labels",
+		},
+	},
+	Action: actionDecorator(labelTransaction),
+}
+
+func labelTransaction(ctx *cli.Context) error {
+	// Display the command's help message if we do not have the expected
+	// number of arguments/flags.
+	if ctx.NArg() != 1 {
```

Comment:
This should be 2 since the label is required? So it'd move from being a flag to an argument.